lib/board.rb

```diff
@@ -0,0 +1,43 @@
+class Board
+  attr_accessor :cells
+
+  def reset!
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def initialize
+    self.reset!
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(user_input)
+    self.cells[user_input.to_i - 1]
+  end
+
+  def full?
+    self.cells.include?(" ") ? false : true
+  end
+
+  def turn_count
+    9 - self.cells.count(" ")
+  end
+
+  def taken?(user_input)
+    self.position(user_input) != " "
+  end
+
+  def valid_move?(user_input)
+    user_input.to_i.between?(1, 9) && taken?(user_input) == false
```

Comment:
Use the implicit boolean return value of a method rather than checking for equality to true or false. 
```
user_input.to_i.between?(1,9) && !taken?(user_input)
```